src/main/java/frc/robot/subsystems/AimingSubsystem.java

```diff
@@ -0,0 +1,244 @@
+  
+    PositionVoltage wristVoltage = new PositionVoltage(getDesiredWristRotation()).withSlot(0);
+    mWristMotor.setControl(wristVoltage.withPosition(getDesiredWristRotation()));
+    mCurrentWristRotationDeg = mWristMotor.getRotorPosition().getValueAsDouble() * 360;
+  }
+
+  private void updateMotorModes() {
+    SmartDashboard.updateValues();
+
+    AimingMotorMode mode = mChooser.getSelected();
+    
+    // Motors go towards setpoints
+    NeutralModeValue neutralmode = mode == AimingMotorMode.BRAKE ? NeutralModeValue.Brake : NeutralModeValue.Coast;
+
+    mLeftFlywheelMotor.setNeutralMode(neutralmode);
+    mRightFlywheelMotor.setNeutralMode(neutralmode);
+    mWristMotor.setNeutralMode(neutralmode);
+
+  }
+
+  // Contains Smart Dashboard Statements ONLY ON DEBUG
+  private void debugSmartDashboard() {
+
+    // TODO: Add debug statements
+  }
+
+  // Getters and Setters of TOF Sensor
+  /**
+   * 
+   * @return range in millimeters
+   */
+  public double getTOFRangeRaw() {
+    return mElevatorTOF.getRange() - 30.0;
+  }
+
+  public double getTOFRangeCentimeters() {
+    return getTOFRangeRaw()/10;
+  }
+
+  public double getTOFRangeInches() {
+    return Math.round(((getTOFRangeRaw())/(25.4))*100.0)/100.0;
+  }
+
+  public void setTOFViewZone(int topLeftX, int topLeftY, int bottomRightX, int bottomRightY) {
+    mElevatorTOF.setRangeOfInterest(topLeftX, topLeftY, bottomRightX, bottomRightY);
+  }
+
+  public void setTOFRange(String sMode, double sampleTime) {
+    switch(sMode.toLowerCase()) {
+      case "short": //Use short only if less than 1.3 meters off the ground and bright light
+        mElevatorTOF.setRangingMode(RangingMode.Short, sampleTime);
+      case "long": //Use for dark lighting up to 4 meters
+        mElevatorTOF.setRangingMode(RangingMode.Long, sampleTime);
+      case "medium": //Use medium if there could potentially be dark lighting
+        mElevatorTOF.setRangingMode(RangingMode.Medium, sampleTime);
+
+    }
+  }
+
+  // Getters and Setters of States
+  public double getCurrentElevatorDistance() {
+    return mCurrentElevatorDistanceIn;
+  }
+
+  public double getDesiredElevatorDistance() {
+    return mDesiredElevatorDistanceIn;
+  }
+  
+  public double getDesiredWristRotation() {
+    return mDesiredWristRotationDeg;
+  }
+
+  public void setDesiredElevatorDistance(double distance) {
+    mDesiredElevatorDistanceIn = distance;
+  }
+
+  public void setDesiredWristRotation(double rotation) {
+    mDesiredWristRotationDeg = rotation;
+  }
+
+  public void setDesiredSetpoint(AimState state) {
+    mDesiredElevatorDistanceIn = state.elevatorDistIn;
+    mDesiredWristRotationDeg = state.wristAngleDeg;
+  }
+  public void changeDesiredElevatorPosition(double rate) {
+    mDesiredElevatorDistanceIn += rate;
+  }
+
+  public void changeDesiredWristRotation(double rate) {
+    mDesiredWristRotationDeg += rate;
```

Comment:
This should be rate*loopTime

Reply:
I'm a little confused. Is looptime a constant i need to add?